src/main/kotlin/com/sourcegraph/cody/agent/protocol_helpers/ClientInfo.kt

```diff
@@ -0,0 +1,25 @@
+package com.sourcegraph.cody.agent.protocol_helpers
+
+import com.sourcegraph.cody.agent.protocol_generated.ClientInfo as ProtocolClientInfo
+
+data class ClientInfo(
```

Comment:
@RXminuS I think it is a bit confusing that helper classes have the same name as the generated one. I feel like a bit idiomatic for this use case would be to simply have a construction method in the `protocol_generated.ClientInfo` extension.

Reply:
Everything in protocl_generated is not allowed to be modified and also contains headers to warn against doing so. If we are missing helpers that we can make an issue to extend the generation system. 

 But I understand what you mean about the naming duplication being confusing. That should mostly go away though as we either use the new protocol classes directly. And for the other 10% of cases I think we're probably best going with a Factory design pattern instead of modifications to classes or wrapper classes.